plan.py

```diff
@@ -33,12 +47,100 @@
 saver = tf.train.Saver()
 saver.restore(sess, 'model/model.ckpt')
 
+if args.use_openvino:
+    import subprocess
+    import sys
+    import mo_tf
+    import os
+
+    from collections import namedtuple
+    from openvino.inference_engine import IECore
+    ie = IECore()
+
+    # Get input nodes names
+    exp_inp_node_x_name = expansion_net.X.name.split(':')[0]
+    exp_inp_node_k_name = expansion_net.k.name.split(':')[0]
+    roll_inp_node_name = rollout_net.X.name.split(':')[0]
+
+    pb_model_path = 'model.pb'
+    model_xml = 'model.xml'
+    model_bin = 'model.bin'
+
+    # EXPANSION_NET:
+    # Save frozen graph
+    input_node_names = [exp_inp_node_x_name, exp_inp_node_k_name]
+    output_node_names = ['TopKV2']
+
+    graph_def = tf.graph_util.convert_variables_to_constants(
+            sess,
+            sess.graph_def,
+            output_node_names)
+
+    from tensorflow.python.tools import optimize_for_inference_lib
+    graph_def = optimize_for_inference_lib.optimize_for_inference(
+            graph_def, input_node_names, output_node_names, tf.float32.as_datatype_enum)
```

Comment:
Will code work without `optimize_for_inference` step?

Reply:
Yes, should I remove it?

Reply:
Yes, if we can work without it